pegasus/sites.v3/hourofcode.com/styles/050-theme.css

```diff
@@ -18,7 +18,7 @@ body {
 
 #pageheader {
   width: 100%; 
-  background-color: #000;
+  background-color: rgba(0, 178, 192, 0.5);
```

Comment:
note that rgba isn't supported on IE8.. worth testing to see what happens in that case.